dependencies improvmemetns (#872)
bruin-data/bruin: docs/assets/definition-schema.md

Must consist of letters and dot `.` character.
- **Type:** `String`

## `uri`

> We use `uri` (Universal resource locator) as another way to identify assets. This should be unique accross all your pipelines and can be used to define [cross pipeline dependencies](../cloud/cross-pipeline).

Reviewer: ```suggestion
We use `uri` (Universal Resource Identifier) as another way to identify assets. URIs must be unique across all your pipelines and can be used to define [cross pipeline dependencies](../cloud/cross-pipeline).
```
